.github/workflows/MarkdownLinksCheck.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Markdown links
```

Comment:
This action checks external links (e.g. `http://tralala.com`) and links to relative files (e.g. `/images/tralala.png`) but not the internal links withing an md-document (e.g. `#5-references`)